src/xcode/ENA/ENA/Source/Scenes/TraceLocations/Overview/TraceLocationsOverviewViewController.swift

```diff
@@ -234,7 +234,17 @@ class TraceLocationsOverviewViewController: UITableViewController, FooterViewHan
 	}
 
 	private func updateEmptyState() {
-		tableView.backgroundView = viewModel.isEmpty ? EmptyStateView(viewModel: TraceLocationsOverviewEmptyStateViewModel()) : nil
+		let emptyStateView = EmptyStateView(viewModel: TraceLocationsOverviewEmptyStateViewModel())
+
+		emptyStateView.additionalTopPadding = tableView.rectForRow(at: IndexPath(row: 0, section: 0)).maxY
```

Comment:
Can you add some comments on why and what is happnening here?

Reply:
Done